ros_buildfarm/doc_job.py

```diff
@@ -427,24 +427,26 @@ def _get_doc_independent_job_config(
         'notify_emails': build_file.notify_emails,
 
         'timeout_minutes': build_file.jenkins_job_timeout,
+
+        'upload_user': build_file.upload_user,
+        'upload_host': build_file.upload_host,
+        'upload_root': build_file.upload_root,
+
+        'credential_id': build_file.upload_credential_id
```

Comment:
I haven't done a commit history dive to check whether accident or intent had the credential variable inserted based on specific job types but if we're going to normalize it I would prefer to keep `upload_credential_id` so that a grep through the codebase turns up more direct hits.

If updating the make_target job is too cumbersome for this patch I would be willing to write one which does so separately.

Reply:
I arbitrarily chose `credential_id`, but I'm happy to switch it to `upload_credential_id`.  It is straightforward to do.  I'll go ahead and do that.